client/src/components/PeriodicTable/ElementInfo.js

```diff
@@ -0,0 +1,831 @@
+
+
+export const ElementTypes = {
+    metal: "metal",
+    metalloid: "metalloid",
+    nonmetal: "nonmetal"
+}
+
+// Source from https://www.angelo.edu/faculty/kboudrea/periodic/structure_numbers.htm
```

Comment:
try to use this link https://pubchem.ncbi.nlm.nih.gov/periodic-table/  to fill out elements 113 - 118.